.dockerignore

```diff
@@ -20,6 +20,13 @@ packages/stat-logger
 **/*.egg-info
 **/swingset-kernel-state
 **/.cache
+**/.pnp.*
```

Comment:
Could you also add `**/.DS_Store` which I had forgotten?

Reply:
(If not here, I can do it in a follow-up.)

Reply:
A yeah mac user!